plugins/org.locationtech.udig.catalog.jgrass.tests/test-data/jgrass-test/newmapset/hist/ArcGrid

```diff
@@ -0,0 +1,8 @@
+Sat Oct 02 16:38:04 CEST 2021
+ArcGrid
+C:\dev\data\geodata\testdata\jgrass-db\jgrass-test\newmapset
+fgdrf
+DCELL
```

Comment:
@moovida  How can I avoid having local pathes and user-name in this file, is it just a comment I can remove? Are these lines required somehow?

Reply:
Removed content and test is still fine